chrome/host.json

```diff
@@ -4,6 +4,6 @@
   "path": "%%replace%%",
   "type": "stdio",
   "allowed_origins": [
-    "chrome-extension://jegbgfamcgeocbfeebacnkociplhmfbk/"
+    "chrome-extension://gkndoddhagggkjfkojkmnimeafhmpeod/"
```

Comment:
Yeah I noticed that, I'm working another issue with the extension ID (#16) so no worries, this'll change before publishing the next release. :)